app/views/content/train-to-be-a-teacher/other-training-you-can-get.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Other training and experience you can get
+related_content:
+    What is QTS?: "/what-is-qts"
+    Salaries and benefits : "/salaries-and-benefits"
+    Online Q&A sessions : "/event-categories/online-q-as"
+---
+
+##Get School Experience
```

Comment:
Are we planning to replace the GSE page altogether? Or just signpost from here?

Reply:
Ah - I see this is content is all from the Get School Experience page but it is not titled as GSE

@valentine-carter I think we might need to do some more thinking around this - this feels a little like we've downgraded Get School Experience which I don't think is right

We may need to think about splitting out GSE onto one card/page and internships onto another (and then apprenticeships onto another when we have some more content)